home/tests/test_whatsapp_template_import_export.py

```diff
@@ -0,0 +1,372 @@
+import csv
+import json
+import re
+from collections.abc import Callable, Iterable
+from dataclasses import dataclass
+from functools import wraps
+from io import BytesIO, StringIO
+from pathlib import Path
+from queue import Queue
+from typing import Any, TypeVar
+
+import pytest
+from django.core import serializers  # type: ignore
+from openpyxl import load_workbook
+from pytest_django.fixtures import SettingsWrapper
+from wagtail.models import Locale  # type: ignore
+from wagtail.snippets.models import register_snippet  # type: ignore
+
+from home.import_whatsapp_templates import ImportWhatsAppTemplateException
+from home.models import (
+    WhatsAppTemplate,
+)
+from home.wagtail_hooks import WhatsAppTemplateAdmin
+from home.whatsapp_template_import_export import import_whatsapptemplate
+
+TTemplate = TypeVar("TTemplate", bound=WhatsAppTemplate)
+
+IMP_EXP_DATA_BASE = Path("home/tests/import-export-data")
+
+ExpDict = dict[str, Any]
+ExpPair = tuple[ExpDict, ExpDict]
+ExpDicts = Iterable[ExpDict]
+ExpDictsPair = tuple[ExpDicts, ExpDicts]
+
+
+def filter_both(
+    filter_func: Callable[[ExpDict], ExpDict]
+) -> Callable[[ExpDict, ExpDict], ExpPair]:
+    @wraps(filter_func)
+    def ff(src: ExpDict, dst: ExpDict) -> ExpPair:
+        return filter_func(src), filter_func(dst)
+
+    return ff
+
+
+@filter_both
+def strip_leading_whitespace(entry: ExpDict) -> ExpDict:
+    # FIXME: Do we expect imported content to have leading spaces removed?
+    bodies = {k: v.lstrip(" ") for k, v in entry.items() if k.endswith("_body")}
```

Comment:
We don't have any fields in templates that end with `_body`, so I don't think this filter will do anything, so maybe we don't need it?